src/tasks/publish/index.ts

```diff
@@ -110,10 +110,11 @@ async function publishTask(
   const distPath = aragonConfig.appBuildOutputPath as string
   const ignoreFilesPath = aragonConfig.ignoreFilesPath as string
   const selectedNetwork = bre.network.name
-  const ipfsGateway = (bre.config.ipfs || {}).ipfsGateway || defaultIpfsGateway
-
+  const ipfsApiUrl = ipfsApiUrlArg || (bre.config.ipfs || {}).ipfsApi
   // TODO: Warn the user their metadata files (e.g. appName) are not correct.
 
+  if (!ipfsApiUrl) throw new BuidlerPluginError(`ipfsApiUrl must be defined`)
```

Comment:
In which case we throw here? Because we have an `http://localhost:5001` as default

Reply:
Fixed